LICENSE

```diff
@@ -1,7 +1,7 @@
 Node.js is licensed for use as follows:
 
 """
-Copyright Node.js contributors. All rights reserved.
+Copyright Joyent, Inc. and other Node.js contributors. All rights reserved.
```

Comment:
Consider this change a WIP and I'm waiting to hear back from the lawyers to let me know if it's adequate :-)